src/components/SearchForm.tsx

```diff
@@ -8,20 +8,23 @@ interface SearchFormProps {
 
 const SearchForm: React.FC<SearchFormProps> = ({initialSearch, onSearch}) => {
     return (
-        <form
-            onSubmit={(e) => {
-                e.preventDefault();
-                const form = e.target as HTMLFormElement;
-                const input = form.elements.namedItem("searchInput") as HTMLInputElement;
-                onSearch(input.value);
-            }}>
-            <input name="searchInput"
-                   className={styles.searchFormInput}
-                   type="text"
-                   autoComplete="none"
-                   placeholder={initialSearch}/>
-            <button type="submit" className={styles.searchFormButton}>Search</button>
-        </form>
+        <div className={styles.searchFormContainer}>
+            <h1 className={styles.formTittle}>Find your movie</h1>
+            <form
+                onSubmit={(e) => {
```

Comment:
Specify type here (e: ...) and you do not need to do assertions 
![image](https://github.com/user-attachments/assets/3641e84b-511e-493b-b9bf-db60f535601d)

Reply:
I got it. Yes because you are using <form> for receiving value. In onChange it will be easy, I think. Leave as is.